Makefile

```diff
@@ -33,6 +41,13 @@ spitfire/runtime/_baked.so spitfire/runtime/_template.so spitfire/runtime/_udn.s
 	$(PIP) install --user --editable .
 
 
+fix:
+	$(YAPF) --in-place --recursive --verify spitfire scripts || (echo; echo 'Files were auto-formatted. Please double-check the changes made.')
```

Comment:
Unfortunately, make runs the `echo` command, not the bash built-in, and doesn't support the `-e` flag.  I adjusted the output to make it a little clearer.